Examples/Scotty-tvOSExample/View Controllers/LeftViewController.swift

```diff
@@ -0,0 +1,30 @@
+//
```

Comment:
Similar comment to above about extracting this into a shared Examples/Common folder and then making it a member of both tvOS and iOS targets. I see this is named `LeftViewController`, while the one in the iOS example is just named `ViewController.swift` so we'd need to rename it.